backend/src/main/routes/EventRoute.ts

```diff
@@ -78,7 +78,6 @@ export default class EventRoute extends CommonRoutesConfig {
 
     this.getApp()
       .route(`/event/:id/accept`)
-      .all(checkJwt)
```

Comment:
Why are we removing the token check for this endpoint call?

Reply:
We do not need to login when accepting an invite. The link is sent to the email. Its the same principal as when we accept the vaccination appointment with clicsante.

Reply:
That makes sense. The rest looks good!